RSFormView/Classes/Models/FormConfigurator.swift

```diff
@@ -21,6 +21,13 @@ public class FormConfigurator {
   public var textColor = UIColor.brightGray
   public var errorTextColor = UIColor.red
   
+  public var validBorderColor = UIColor.lightGray
+  public var invalidBorderColor = UIColor.red
+  public var editingBorderColor = UIColor.astralBlue.withAlphaComponent(0.8)
```

Comment:
Lets default the borders to clear color, and have the example app set up this colors

Reply:
working on that right now 😉